pontoon/messaging/static/js/messaging.js

```diff
@@ -162,13 +163,87 @@ $(function () {
     $('#review .message-type .transactional').toggle(isTransactional);
   }
 
+  function loadPanelContent(path) {
+    if (inProgress) {
+      inProgress.abort();
+    }
+
+    const panel = container.find('.right-column');
+    const relative_path = path.split('/messaging/')[1];
+    const menu_path = path === '/messaging/sent/' ? path : '/messaging/';
+
+    // Update menu selected state
+    container
+      .find(`.left-column a[href="${menu_path}"]`)
+      .parents('li')
+      .addClass('selected')
+      .siblings()
+      .removeClass('selected');
+
+    panel.empty();
+
+    inProgress = $.ajax({
+      url: `/messaging/ajax/${relative_path}`,
+      success: function (data) {
```

Comment:
Is there a reason to persist with this verbose style?
```suggestion
      success(data) {
```